### PR TITLE
Fix CoMD .good file

### PR DESCRIPTION
**Pull request by benharsh**

### CODE REVIEW

**Comment by benharsh** [comment]
@ben-albrecht : can you review?

File: test/studies/comd/llnl/CoMD.prediff [hunk 1]
```diff
@@ -6,17 +6,36 @@ test_name = sys.argv[1]
...
+vals['   Initial energy  :'] = -1.166063303477
+vals['   Final energy    :'] = -1.166049767266
+vals['   eFinal/eInitial :'] = 0.999988
+
+def verify(line):
+    info = line.split(' ')
+    pre = ' '.join(info[:-1])
+    if pre in vals:
+        last = float(info[-1])
+        if abs(last - vals[pre]) < EPSILON:
```
Comment (ben-albrecht) [inline test/studies/comd/llnl/CoMD.prediff]: Could alternatively use [`math.isclose()`](https://docs.python.org/3/library/math.html#math.isclose)

Comment (benharsh) [inline test/studies/comd/llnl/CoMD.prediff]: Ah, interesting. I don't think we can rely on python 3 for start_test yet though 😢

Comment (ben-albrecht) [inline test/studies/comd/llnl/CoMD.prediff]: Good point - didn't even realize it's python3-only..

**Pull request merged**